views/tag/search_results_modal.html

```
@@ -101,7 +101,7 @@ <h3 class="text-lg font-medium text-gray-900 dark:text-white">{{.Title}}</h3>
                 </a>
                 {{end}}
                  <button
-                   hx-get="/ui/tag/edit/{{$.TrackID}}/select/{{$.ProviderName}}?index={{$index}}"
+                   hx-get="/tag/{{$.TrackID}}/select/{{$.ProviderName}}?index={{$index}}"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)